templates/zerver/help/edit-or-delete-a-message.md

```diff
@@ -29,15 +29,11 @@ content.
 
     **Note:** After you have edited a message, the message is publicly
     marked as **EDITED**. You can
-    [view a message's edit history](/help/view-a-messages-edit-history),
-    assuming that feature has not been
-    [disabled by an organization administrator](/help/disable-message-edit-history).
+    [view a message's edit history](/help/view-a-messages-edit-history)
+    if it is [enabled](/help/disable-message-edit-history) in your organization.
 
-If you don't see the pencil (<i class="fa fa-pencil"></i>) icon, the message content
-can no longer be edited. You should see a file (<i class="fa fa-file-code-o"></i>)
-icon instead. Clicking the file icon will allow you to view the
-[Markdown source](/help/view-the-markdown-source-of-a-message) of the message, or
-[edit the topic](/help/rename-a-topic).
+If you don't see the **pencil** (<i class="fa fa-pencil"></i>) icon, the content
+of your message can no longer be edited.
```

Comment:
Let's move this into a warning block at the bottom of the instructions list, with a wording change, since it's possible the messages was never editable to begin with.

> If you don't see the **pencil** (<i class="fa fa-pencil"></i>) icon, you are not permitted to edit this message.